shanoir-ng-datasets/src/main/java/org/shanoir/ng/datasetacquisition/service/DatasetAcquisitionServiceImpl.java

```diff
@@ -2,12 +2,12 @@
  * Shanoir NG - Import, manage and share neuroimaging data
  * Copyright (C) 2009-2019 Inria - https://www.inria.fr/
  * Contact us on https://project.inria.fr/shanoir/
- * 
+ * <p>
```

Comment:
Sorry but could you remove those ? Are those for the java doc ? Let's discuss it in the daily (we have a way to update every header at once).